packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -176,14 +175,40 @@ class RunPanel extends React.PureComponent {
                     {content}
                 </Drawer>
                 {is_mobile && <MobileDrawerFooter {...this.props} should_show_statistics={active_index !== 2} />}
-                <Dialog
+                <Modal
```

Comment:
Could we use components' `<Dialog>` instead? 🙈

Reply:
@aaron-binary  as I talked with you  Dialog component has not close icon according to design we have close icon and because of that I use Modal if we decide to use Dialog component I can update it and add close icon